run_api_background.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Run API server in background for testing
+"""
+
+import subprocess
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)